### PR TITLE
Halo2 GWC benchmark

### PR DESCRIPTION
**Pull request by tyshko-rostyslav**

### CODE REVIEW

File: halo2-gwc/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+                message_arr: [Fr::ZERO; L],
+                message: Value::unknown(),
+                _spec: PhantomData,
+            }
+        }
+
+        fn configure(meta: &mut ConstraintSystem<Fr>) -> Self::Config {
+            let state = (0..WIDTH).map(|_| meta.advice_column()).collect::<Vec<_>>();
+            let _ = meta.instance_column();
+            // // meta.enable_equality(expected);
```
Comment (jonesmarvin8) [inline halo2-gwc/src/main.rs:76]: Line 76 can be removed since commented out.

**Comment by jonesmarvin8** [comment]
Outside of the commented out code. I think just crediting portion of PSE code in a comment is all the remains.

Everything looks good.

File: halo2-gwc/src/main.rs (hunk 1 above)
Comment (tyshko-rostyslav) [reply halo2-gwc/src/main.rs]: Done

**Review by jonesmarvin8 — APPROVED** [review]
Looks good.

**Pull request merged**